stablehlo/tests/stablehlo_canonicalize_dynamism.mlir

```diff
@@ -534,11 +522,7 @@ func.func @real_dynamic_slice_to_slice_success_static_result_type(%arg0: tensor<
 // CHECK-LABEL: func @real_dynamic_slice_to_slice_success_dynamic_result_type
 func.func @real_dynamic_slice_to_slice_success_dynamic_result_type(%arg0: tensor<4xf32>) -> tensor<?xf32> {
   //  CHECK-NOT: stablehlo.real_dynamic_slice
-  //      CHECK: "stablehlo.slice"(%arg0) {
-  // CHECK-SAME:   limit_indices = dense<1> : tensor<1xi64>,
-  // CHECK-SAME:   start_indices = dense<0> : tensor<1xi64>,
-  // CHECK-SAME:   strides = dense<2> : tensor<1xi64>
-  // CHECK-SAME: } : (tensor<4xf32>) -> tensor<?xf32>
+  //      CHECK: stablehlo.slice %arg0 [0:1:2]
```

Comment:
I like the syntax - it matches Python and is close to HLO. Also cc @GleasonK since he may have had some thoughts about prettyprinting for slicing as well.